faucet.go

```diff
@@ -198,10 +85,9 @@ type lightningFaucet struct {
 
 	templates *template.Template
 
-	openChanMtx             sync.RWMutex
-	openChannels            map[wire.OutPoint]time.Time
-	disableGenerateInvoices bool
-	disablePayInvoices      bool
+	rateLimitMtx sync.RWMutex
```

Comment:
This mutex should be defined along side the var it is protecting (`requestIPs`) rather than inside of the `lightningFaucet` struct